### ISSUE TITLE
Handle DB tables on uninstall

### ISSUE THREAD

**Issue by Eldenroot** [MattRogowski/MyReactions#28]
Is there any protection mechanism to not lose DB table with given reactions on uninstall? If not, can be add any?

**Comment by MattRogowski** [comment]
Hm, uninstalling is really meant to remove everything - on some plugins I have a confirmation when you install, I could add that with an option to export the data first?

**Comment by Eldenroot** [comment]
In Thank you like/plugin there was a lock file - which must be deleted first to delete all your data from DB. It has been replaced by a confirmation dialog on uninstall.

Maybe you can use one of these approaches.

https://github.com/Cu8eR/MyBB_Thank-you-like-plugin/commit/51792a1210539a6cfc8b98fa05a0ca51cb386911

EDIT:
It would be better than lose everything on missclick

**Comment by MattRogowski** [comment]
Yeah I'll put something in, have always thought the current system is pretty risky. A lock file is an interesting idea, but in MySupport I have a confirmation page so will probably do the same.
